.circleci/config.yml

```diff
@@ -412,31 +412,41 @@ jobs:
             echo ${GCLOUD_SERVICE_KEY} | docker login -u _json_key --password-stdin https://gcr.io
 
       - docker-cache/with-save-restore-images:
+         parallelism: 8
          repository: gcr.io/${GOOGLE_PROJECT_ID}
          images: >-
            ${IMAGE}-cache:${DOCKER_BRANCH_TAG}
            ${IMAGE}-cache:latest
-           ${IMAGE}-develop:latest
+           #${IMAGE}-develop:latest
```

Comment:
`#` doesn't work here. Just remove the line.

```
Error parsing reference: "#taraxa-node-develop:latest" is not a valid repository/tag: invalid reference format
```

Reply:
No, yes, it is being built, it is just not cached (saved/pushed as cache)